includes/class-wcpdf-install.php

```diff
@@ -255,6 +255,7 @@ protected function upgrade( $installed_version ) {
 					'currency_font'				=> array( 'wpo_wcpdf_template_settings' => 'currency_font' ),
 					'paper_size'				=> array( 'wpo_wcpdf_template_settings' => 'paper_size' ),
 					'header_logo'				=> array( 'wpo_wcpdf_template_settings' => 'header_logo' ),
+					'header_logo_height'		=> array( 'wpo_wcpdf_template_settings' => 'header_logo_height' ),
```

Comment:
This is the settings migration routine, but this setting wasn't there in 1.X so nothing to migrate here! This can be omitted